tests/php/Controllers/Tabbar/TabbarControllerTopActionTest.php

```diff
@@ -0,0 +1,346 @@
+<?php
+
+use App\Http\Controllers\Tabbar\TabbarController;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\HomeUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\ContentsUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\DocsUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\DevUrlInfo;
+use Illuminate\Http\Request;
+
+/**
+ * TabbarControllerクラスのtabbarActionのテストを行う.
+ *
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class TabbarControllerTabbarActionTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)